website/frontend/templates/quick-start.html

```diff
@@ -9,6 +9,10 @@
         This quick start guide describes how you can tag your files using
         <a href="/" title="MusicBrainz Picard">MusicBrainz Picard</a>.
       </p>
+      <p>
+        To make things clear - picard is not built to be a mass-single-track/recording tagger/tag fixer.
```

Comment:
s/picard/Picard. Also maybe we would like to tone down the warning a bit. It sounds too upfront.

Reply:
Since this hasn't seen any changes in a while, can I finish this up? 
I want to get started contributing here
@samj1912